org.gnomeradio.Radio.appdata.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<application>
+  <id type="desktop">org.gnome.Radio.desktop</id>
+  <name>GNOME Radio 64 for GNOME 44</name>
+  <summary>Listen to National Public Radio on GNOME 44</summary>
+  <licence>CC0</licence>
+  <description>
+    <p>
+      GNOME Radio 64 is a Free Software program that allows you easily
+      listen to National Public Radio (NPR) broadcasts under GNOME 44.
+    </p>
+    <p>
+      GNOME Radio 64 is developed on the GNOME 44 desktop platform and
+      it requires GTK4, Clutter and GStreamer 1.0 for playback.
+    </p>
+    <p>    
+    Enjoy National Public Radio (NPR) broadcasts under GNOME 44.
```

Comment:
`GNOME 44` is repeated a bit too many times to make this a good description.

Reply:
Fixed.